.github/workflows/action-test-development.yml

```diff
@@ -0,0 +1,60 @@
+# Action that runs the full development stack:
+# Steps (Python-only):
+# - creates esmvaltool conda environment and pip-installs esmvaltool
+# - downloads latest master of esmvalcore and installs it in development mode
+# - runs tests of esmvaltool
+# Triggered by a push to master and nightly
+
+name: Develop Test
+
+# runs on a push on master and at the end of every day
+on:
+  push:
+    branches:
+    - master
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Did you want to run this at 4 am instead, like the other jobs?

Reply:
errmm no, maybe, no...doesn't really matter coz it's not pulling the package of `esmvalgroup` channel - we had issues with that at exactly midnight, for some odd reason